app/seeds/__init__.py

```diff
@@ -1,6 +1,7 @@
 from flask.cli import AppGroup
 from .users import seed_users, undo_users
-
+from .states import seed_states, undo_states
+from .locations import seed_cities
```

Comment:
do we need to import undo_cities here? I saw that you created it in app/seeds/locations line 28